index.ts

```diff
@@ -60,3 +60,59 @@ test('test migration run', async() => {
 
     assert(ahAccAfter.free.eq(rcAccBefore.free));
 });
+
+test('test migration', async() => {
+    const {polkadot, assetHub} = await setupNetworks({
+        polkadot: {
+            endpoint: process.env.POLKADOT_ENDPOINT || 'ws://localhost:9944',
+            // resumes from the highest block.
+            // resume: true,
+            'wasm-override': 'runtime_wasm/polkadot_runtime.compact.compressed.wasm',
+            // 'runtime-log-level': 5,
+            db: './dbs/polkadot.sqlite',
+            port: 8000,
+        },
+        assetHub: {
+            endpoint: process.env.POLKADOT_ASSET_HUB_ENDPOINT || 'ws://localhost:9945',
+            // resumes from the highest block.
+            // resume: true,
+            'wasm-override': 'runtime_wasm/asset_hub_polkadot_runtime.compact.compressed.wasm',
+            // 'prefetch-storages': ['0x'],
+            // 'runtime-log-level': 5,
+            db: './dbs/polkadot-asset-hub.sqlite',
+            port: 8001,
+        },
+    });
+
+    await polkadot.dev.setStorage({
+        rcMigrator: {
+            rcMigrationStage: 'AccountsMigrationInit',
+        },
+    })
+
+    await assetHub.dev.setStorage({
+        ahMigrator: {
+            ahMigrationStage: 'DataMigrationOngoing',
+        },
+    });
+
+    await polkadot.dev.newBlock(); // Init -> Ongoing
+    await assetHub.dev.newBlock();
+
+
+    let rcMigrationStage = (await polkadot.api.query.rcMigrator.rcMigrationStage()).toHuman();
+    let rcStageName = rcMigrationStage ? Object.keys(rcMigrationStage)[0] : null;
+
+    console.log('rcMigrationStage before: ', rcMigrationStage);
+    while (rcStageName == 'AccountsMigrationOngoing') {
```

Comment:
`==` is a code smell in TS/JS: it performs type coercion (under a complicated set of rules that may be applied more than once to each operand), and may lead to unexpected results.
When in doubt, use `===`.

Also, TypeScript has enums like Rust's! It would not be bad to use them here over strings.